ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -17,9 +17,28 @@
 import numpy as np
 from tqdm import tqdm
 from grpc._channel import _InactiveRpcError, _MultiThreadedRendezvous
-from ansys.api.mapdl.v0 import mapdl_pb2 as pb_types
-from ansys.api.mapdl.v0 import mapdl_pb2_grpc as mapdl_grpc
-from ansys.api.mapdl.v0 import ansys_kernel_pb2 as anskernel
+
+try:
+    from ansys.api.mapdl.v0 import mapdl_pb2 as pb_types
+    from ansys.api.mapdl.v0 import mapdl_pb2_grpc as mapdl_grpc
+    from ansys.api.mapdl.v0 import ansys_kernel_pb2 as anskernel
+
+except ImportError:
```

Comment:
With multiline error strings, use the approach in:
https://github.com/pyansys/pymapdl/blob/4566ef4a525e02b88f6e42f66d832286269c25af/ansys/mapdl/core/errors.py#L14-L21

Reply:
Done.